src/coprocessor/dag/expr/builtin_string.rs

```diff
@@ -1854,6 +1902,77 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_field_string() {
```

Comment:
Please add more tests for `field_int` and `field_real`.

Reply:
@huachaohuang Done.